### DISCUSSION TITLE
CDSE S3 endpoint

### DISCUSSION THREAD

**Discussion by pl-marasco** [esa-cdab/cdab-testsuite#8] [Q&A] [answered]
How can tests be conducted using the S3 bucket backend? 
I would like to start using this platform for benchmark, but I'm not able to find the documentation on how to use the S3 bucket. 

Meanwhile, I had to implement this [CDSE_Benchmark](https://github.com/pl-marasco/CDSE_Benchmark) and, if you would like, feel free to take inspiration from it. We have been testing a few user end points  and this brought us to the necessity of using an "agreed" approach.

**Comment by esa-cdab** [comment, marked as answer]
Hello pl-marasco,
the settings to configure data access with S3 are defined here:
https://github.com/esacdab/cdab-testsuite/wiki/Configure-The-Test-Suite
